ironfish/src/metrics/metricsMonitor.ts

```diff
@@ -66,6 +67,7 @@ export class MetricsMonitor {
 
     this.mining_newBlockTemplate = this.addMeter()
     this.chain_newBlock = this.addMeter()
+    this.chain_databaseSize = new Gauge()
```

Comment:
We should probably find a way to initialize this to the size of the chain db. Maybe set it when the database starts up so that we don't get 0 values sent to telemetry if it's not initialized

Reply:
Added in the blockchain.open()